src/serena/config/serena_config.py

```diff
@@ -342,6 +342,8 @@ class SerenaConfig(ToolInclusionDefinition, ToStringMixin):
     trace_lsp_communication: bool = False
     web_dashboard: bool = True
     web_dashboard_open_on_launch: bool = True
+    web_dashboard_host: str = "127.0.0.1"
```

Comment:
Calling this a "host" is misleading. We should prefer the more accurate term `listen_address`.